internal/xds/bootstrap/bootstrap.go

```diff
@@ -204,3 +208,10 @@ func GetRenderedBootstrapConfig(proxyMetrics *egv1a1.ProxyMetrics) (string, erro
 
 	return cfg.rendered, nil
 }
+
+func validateRegex(regex string) error {
```

Comment:
can we move this into https://github.com/envoyproxy/gateway/tree/main/internal/utils so it can be reused instead of defining it twice ?